book/src/pil/builtins.md

```diff
@@ -66,6 +66,16 @@ It converts a field element to an integer.
 
 If the argument is already an integer, it is returned without modification.
 
+```rust
```

Comment:
Why rust?

Reply:
To get syntax highlighting, I guess it's the closest syntax